themes/bootstrap3/templates/RecordDriver/DefaultRecord/versions-link.phtml

```diff
@@ -0,0 +1,21 @@
+<?php
+  $versions = $this->driver->tryMethod('getOtherVersionCount');
+  if ($versions):
+?>
```

Comment:
I don't see a reason to close and reopen the php tag here -- probably a side effect of some past changes. Might be worth pulling this all together again.